docs/guides/src/main/server/run-containers.adoc

```diff
@@ -0,0 +1,107 @@
+<#import "/templates/guide.adoc" as tmpl>
```

Comment:
Rename to containers.adoc for nicer link on the website

Reply:
done